docs/nodes/nodes-library/nodes/CrateDB/README.md

```diff
@@ -63,6 +63,11 @@ The start node exists by default when you create a new workflow.
 
 ![Using the CrateDB node to insert data into a table](./CrateDB1_node.png)
 
+## FAQs
+
+### How to specify the data type of a field?
```

Comment:
@harshil1712 Which field are you referring to here? Does that work for all fields, e.g., table, and so on?